docs/learn/learn-account-advanced.md

```diff
@@ -167,6 +168,53 @@ first byte: `02f2d606a67f58fa0b3ad2b556195a0ef905676efd4e3ec62f8fa1b8461355f1142
 counterintuitive that some addresses always have the same prefix and others like Kusama can vary
 wildly, but it's just a quirk of Base58-check encoding.
 
+## System Accounts
+
+
+As the word suggests, system accounts are used by the system. They are used, for example, for the
+treasury, crowdloans, and nomination pools. From the point of view of the runtime, these accounts 
+are like any other account on-chain. These special system accounts are just public keys, with the 
+private key being unknown (and unattainable). So, that means that only the pallet itself can interact 
+with this account. These accounts can never issue a signed [extrinsic](./learn-extrinsics.md) since they 
+do not have a private key.
```

Comment:
I don't understand the obsession with public key cryptography that kind of runs throughout all this. It really distracts from what an account actually is. Accounts are just storage items in the system that are identified by 32-byte keys, and have values that conform to a defined datagram. There are lots of ways to come up with 32-byte keys, like coming up with a word, let's say "py/trsry" and zero-padding it to be 32 bytes. "polkadotpolkadotpolkadotpolkadot" is also 32 bytes (that is 5Ec8KWYgo6B4xTjtu6jo2NZz12ouH1VNqJPcC9rvqiscERnu). You can send DOT to that address. And you can spend it, via governance.

Of course, I can generate a secret, and derive a 32 byte value that is safe to share with others without exposing any information about the secret. And I can generate messages (signatures) using that secret to show that I know the secret from which that value was derived without revealing any information about the secret. Don't get me wrong, this is cool. But irrelevant.

There are lots of ways to create rules about when the value associated with a key can be mutated. Of course one of them is signing messages. Another one is via the very logic that created the key and made it "special". Another is via governance, which can mutate the value behind any key, whether it was generated as the counterpart of a secret, as part of some deterministic logic, or was just nonsense.

Reply:
I am fine with removing the whole account cryptography part. I only tried to make the existing content in the PR more intuitive. Will do that now